Jenkinsfile

```diff
@@ -46,7 +46,7 @@ def sanitized_JOB_NAME = JOB_NAME.toLowerCase().replaceAll('/', '-').replaceAll(
 
 def daos_packages_version = ""
 def el7_component_repos = ""
-def component_repos = ""
+def component_repos = "spdk@PR-11 dpdk@PR-9"
```

Comment:
I presume this needs to be reverted before we land this?

Reply:
Indeed.  @tanabarr is doing a test right now to ensure that when we do land the RPM updates that release/0.9 will not pick them up and will continue to use the existing versions.  Once that test proves, we can land the spdk/dpdk RPMs and then remove the above.